Presentation.Web/Controllers/External/V2/GeneralStakeholderController.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+using Presentation.Web.Models.External.V2;
+using Swashbuckle.Swagger.Annotations;
+
+namespace Presentation.Web.Controllers.External.V2
+{
+    public partial class GeneralController
```

Comment:
Aktørnavnene i controllerne er jeg ikke fan af og især ikke at du bruger partial classes. for at få et route prefix, når det her er en it-system controller

Reply:
Jeg tænker det som use case API'er som følger kalders behov mere end en domænemodel som skal udstilles? Jeg har forsøgt at følge spec hvad angår route prefix

Reply:
Den talte vi om tidlere :-)